composer.json

```diff
@@ -24,12 +24,12 @@
   },
   "suggest": {
     "ext-gmp": "needed for JWT support",
-    "lcobucci/jwt": "needed for JWT support (3.*)",
+    "lcobucci/jwt": "needed for JWT support (3.4.*)",
```

Comment:
Why do you now limit the version pinning to minor versions?

Reply:
I'm not sure at which version the new "lcobucci/jwt" 4.0 compatible functions got available. The deprecation warnings were introduced with 3.4.0 and 3.4 is so far the latest feature version in version 3. I think I can at least adjust the version constraint to >3.4 < 4.0 to cover a greater range if you agree.

Reply:
We could use https://getcomposer.org/doc/articles/versions.md#tilde-version-range- or https://getcomposer.org/doc/articles/versions.md#caret-version-range- so either `~3.4` or `^3.4.0`

Reply:
I'm using ^3.4.0 now